sql/log_event_client.cc

```diff
@@ -1095,6 +1079,42 @@ Rows_log_event::print_verbose_one_row(IO_CACHE *file, table_def *td,
   return 0;
 }
 
+/**
+  Construct a row image according to field information.
+
+  @param[in] cols_bitmap  The bitmap of row columns
+  @param[in] fields       Values of columns used to construct a row image.
+  @buf[out]  buf          Where the row image stored.
+ */
+static uchar *fill_row_image(const MY_BITMAP *cols_bitmap,
+                             const Rows_log_event::Field_info *fields,
+                             uchar *buf)
+{
+  uchar *null_bits= buf;
+  uint nulls_bit_index = 0;
```

Comment:
Really minor thing, this should have no space before the equals

Reply:
fixed